rtsf-at-checkout-loss-detector/res/configuration.toml

```
@@ -6,15 +6,16 @@
 LogLevel = 'INFO'
 
 [Service]
-BootTimeout = '30s'
-ClientMonitor = '15s'
 CheckInterval = '10s'
+HealthCheckInterval = '15s'
```

Sorry, I meant to say  rename `CheckInterval` to `HealthCheckInterval`.
So `CheckInterval` also needs to be removed.
```suggestion
HealthCheckInterval = '10s'
```

---

I have removed  CheckInterval and updated the time duration for HealthCheckInterval